accounting/filters/OsgScheddGpuFilter.py

```diff
@@ -0,0 +1,935 @@
+    10: "All CPU Hours",
+    15: "All GPU Hours",
+    20: "Num Uniq Job Ids",
+    30: "% Good CPU Hours",
+    35: "% Good GPU Hours",
+
+    45: "% Ckpt Able",
+    50: "% Rm'd Jobs",
+    60: "% Short Jobs",
+    70: "% Jobs w/>1 Exec Att",
+    80: "% Jobs w/1+ Holds",
+    81: "% Jobs Over Rqst Disk",
+    82: "% Jobs using S'ty",
+    83: "Total Files Xferd",
+    84: "OSDF Files Xferd",
+    85: "% OSDF Files",
+    86: "% OSDF Bytes",
+
+    88: "Shadw Starts / Job Id",
+    90: "Exec Atts / Shadw Start",
+
+    110: "Min Hrs",
+    120: "25% Hrs",
+    130: "Med Hrs",
+    140: "75% Hrs",
+    145: "95% Hrs",
+    150: "Max Hrs",
+    160: "Mean Hrs",
+    170: "Std Hrs",
+
+    180: "Input Files / Exec Att",
+#    181: "Input MB / Exec Att",
+#    182: "Input MB / File",
+    190: "Output Files / Job",
+#    191: "Output MB / Job",
+#    192: "Output MB / File",
+
+    200: "Max Rqst Mem MB",
+    210: "Med Used Mem MB",
+    220: "Max Used Mem MB",
+    225: "Max Rqst Disk GB",
+    227: "Max Used Disk GB",
+    230: "Max Rqst Cpus",
+    240: "Max Rqst Gpus",
+
+    300: "Good CPU Hours",
+    303: "Good GPU Hours",
+    305: "CPU Hours / Bad Exec Att",
+    307: "GPU Hours / Bad Exec Att",
+    310: "Num Exec Atts",
+    320: "Num Shadw Starts",
+    330: "Num Rm'd Jobs",
+    340: "Num DAG Node Jobs",
+    350: "Num Jobs w/>1 Exec Att",
+    360: "Num Short Jobs",
+    390: "Num Ckpt Able Jobs",
+}
+
+
+DEFAULT_FILTER_ATTRS = [
+    "RemoteWallClockTime",
+    "CommittedTime",
+    "RequestCpus",
+    "RequestGpus",
+    "RequestMemory",
+    "RequestDisk",
+    "RecordTime",
+    "JobStartDate",
+    "JobCurrentStartDate",
+    "MemoryUsage",
+    "DiskUsage",
+    "NumJobStarts",
+    "NumShadowStarts",
+    "NumHolds",
+    "JobUniverse",
+    "JobStatus",
+    "EnteredCurrentStatus",
+    "BytesSent",
+    "BytesRecvd",
+    "TransferInputStats",
+    "TransferOutputStats",
+    "SingularityImage",
+    "ActivationDuration",
+    "ActivationSetupDuration",
+    "CondorVersion",
+]
+
+
+SITE_MAP = get_site_map()
+
+
+class OsgScheddGpuFilter(BaseFilter):
+    name = "OSPool GPU schedd job history"
+
+    def __init__(self, **kwargs):
+        self.collector_hosts = {"cm-1.ospool.osg-htc.org", "cm-2.ospool.osg-htc.org", "flock.opensciencegrid.org"}
+        self.schedd_collector_host_map_pickle = Path("ospool-host-map.pkl")
+        self.schedd_collector_host_map = {}
+        if self.schedd_collector_host_map_pickle.exists():
+            try:
+                self.schedd_collector_host_map = pickle.load(open(self.schedd_collector_host_map_pickle, "rb"))
+            except IOError:
+                pass
+        self.schedd_collector_host_map_checked = set(self.schedd_collector_host_map)
+        # Recheck and update the collector map every Monday during the daily report
+        if kwargs.get("report_period") == "daily" and date.today().weekday() == 0:
+            self.schedd_collector_host_map_checked = set()
```

Comment:
Along with the last suggestion, forgot to suggest getting rid of this as well (sorry!)
```suggestion
```